examples/examplepb/BUILD.bazel

```diff
@@ -49,3 +50,18 @@ go_library(
     embed = [":examplepb_go_proto"],
     importpath = "github.com/grpc-ecosystem/grpc-gateway/examples/examplepb",
 )
+
+protoc_gen_swagger(
+    name = "expamplepb_protoc_gen_swagger",
+    proto_service = "a_bit_of_everything.proto",
```

Comment:
You need to change to `proto = ":examplepb_proto",`. If you do, then you will get a bunch of errors, because the output of the `proto_library` is a list of proto descriptors in `.bin` files. So you will need to use a `Provider` to grab the `.proto` files from the `src` list in the `proto_library`.